javascript-timers/interval.js

```diff
@@ -0,0 +1,15 @@
+var $h1 = document.querySelector('h1');
+
+function countDownEarth() {
+  if ($h1.textContent === '4') {
+    $h1.textContent = '3';
+  } else if ($h1.textContent === '3') {
+    $h1.textContent = '2';
+  } else if ($h1.textContent === '2') {
+    $h1.textContent = '1';
+  } else if ($h1.textContent === '1') {
+    $h1.textContent = '~Earth Beeeelooowww Us~';
+  }
+}
+
+setInterval(countDownEarth, 1000);
```

Comment:
You will need the timer id of the setInterval function. You can capture that with:

`var timerId = setInterval(countDownEarth, 1000);`